model/src/ww3_outp.F90

```diff
@@ -1439,10 +1441,12 @@ SUBROUTINE W3EXPO
             RHOAIR   = MAX ( 0. , DAIRO(J))
 #endif
             CDIR     = MOD ( 270. - CDO(J)*RADE , 360. )
+#ifdef W3_IS2
```

Comment:
Okay- here is the added If def for IS2!   Thanks @benoitp-cmc I just wanted to make sure we had all the code updates and I just missed this on my first pass.